contrib/podmanimage/machine/Containerfile

```diff
@@ -0,0 +1,26 @@
+FROM quay.io/centos-bootc/fedora-bootc:eln
+RUN dnf -y install podman subscription-manager crun crun-wasm crun-krun chrony; dnf -y clean all
+RUN dnf -y install skopeo buildah; dnf -y clean all
+RUN systemctl enable podman.service
+RUN printf "\nconfdir /etc/chrony.d\n" >> /etc/chrony.conf
+
+COPY 10-inotify-instances.conf /usr/etc/sysctl.d/10-inotify-instances.conf
+COPY 50-podman-makestep.conf /usr/etc/chrony.d/50-podman-makestep.conf
+COPY 999-podman-machine.conf /usr/etc/containers/registires.conf.d/999-podman-machine.conf
+COPY containers-etc.conf /usr/etc/containers/containers.conf
+COPY delegate.conf /usr/etc/systemd/system/user@.service.d/delegate.conf
+COPY docker-host.sh /usr/etc/profile.d/docker-host.sh
+COPY podman-docker.conf /usr/etc/tmpfiles.d/podman-docker.conf
```

Comment:
And this should be `/usr/lib/tmpfiles.d` etc.

Reply:
Moved.